### ISSUE TITLE
More Detailed Folder Structure?

### ISSUE THREAD

**Issue by ZeHipo** [Allar/ue4-style-guide#43]
I would like to propose a GUI folder, with sub-folders for e.g. menu, gameplay, cinematics.

While the example folder structure given is probably meant as a rough draft it would be beneficial to make it more definitive and complete, so setting up new projects gets easier for less experienced teams.

**Comment by zaha** [comment]
I agree, that's why I added exactly that in [my forked variant of this style guide](https://github.com/StarforgeGames/ue4-style-guide#structure).

**Comment by Schizo** [comment]
I would love to see as well some more complete folder structures, especially with more complex games.

**Comment by lucastucious** [comment]
We have a dedicated folder to Level Sequence in our game and I think it should be this way.

We name it "Sequences", but maybe we can name it "RTC" for "Real Time Cinematic"
